packages/core/ios/EmbraceManager.swift

```diff
@@ -439,14 +451,19 @@ class EmbraceManager: NSObject {
             attributes["http.response.body.size"] = String(Int(bytesReceived))
         }
 
-        Embrace.client?.recordCompletedSpan(name: createNetworkSpanName(url: url, httpMethod: httpMethod),
-                                            type: SpanType.networkRequest, parent: nil,
-                                            startTime: dateFrom(ms: startInMillis), endTime: dateFrom(ms: endInMillis),
-                                            attributes: attributeStringsFrom(dict: attributes as NSDictionary),
-                                            events: eventsFrom(array: []),
-                                            errorCode: nil
-        )
+        var embraceOtel = EmbraceOTel()
+        var spanBuilder = embraceOtel
```

Comment:
since `spanBuilder` isn't otherwise being used you could do grab the `span` directly:

```
let span =  Embrace.client?.buildSpan(...).setStartTime(...).startSpan()
```